client.go

```diff
@@ -15,11 +15,10 @@ func (p *Provider) setRecord(ctx context.Context, zone string, record libdns.Rec
 	p.mutex.Lock()
 	defer p.mutex.Unlock()
 
-	// there is some strange regexp in the api that prevents us from searching names ending with a dot
-	// so we must ensure the names of the record is relative and does not end with a dot
-	recRelativeName := strings.TrimRight(strings.TrimSuffix(record.Name, domain.Fqdn), ".")
```

Comment:
In fact no we don't have a strange regexp in the api but this is a normal behavior.

When you specify this record in your BIND zone:
```
hello.skusku.pw    IN   A    3600    127.0.0.1
```
The record will be available in `hello.skusku.pw.skusku.pw` because we wasn't added the point to mark the end of the FQDN (also named `absolute record` in the RFC1035 (*Domain names that end in a dot are called
absolute, and are taken as complete*).

I think you already know that, just as reminder ;)